packages/lib/survey/service.ts

```diff
@@ -74,6 +77,16 @@ export const selectSurvey = {
       value: true,
     },
   },
+  // userSegmentId: true,
+  userSegment: {
```

Comment:
As the survey object grows we should also think about if the one service for all surveys architecture makes sense.
This means we are currently also getting all the userSegments in the Survey List in the Formbricks Dashboard as well as the Survey Summary, the Response View and we send it to the formbricks-js SDK. But in all places we don't need the user Segments. I don't have a solution for this yet, but thought it's worth mentioning.